api/views.py

```diff
@@ -54,6 +55,13 @@ class ProfileViewSet(viewsets.ModelViewSet):
     def get_queryset(self):
         return Profile.objects.filter(user=self.request.user)
 
+class UserViewSet(viewsets.ModelViewSet):
+    serializer_class = UserSerializer
+    permission_classes = [permissions.IsAuthenticated, IsOwner]
```

Comment:
We should probably include `http_method_names` like we do in the `ProfileViewSet`, so clients can't delete user objects without going thru site UI.
```
http_method_names = ['get', 'post', 'head', 'put', 'patch']
```

Reply:
Oh hmm, I wonder why I didn't include that in the first place. Fixed in df47f8a2.